### PR TITLE
Chipped 1.18 翻译更新

### PR DESCRIPTION
**Pull request by ChuijkYahus**
- [x] 我已**仔细**阅读贡献指南 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)；
- [x] 我已确认标题符合`{模组英文全名} {简述}`的格式，如`Ender IO 翻译新增/更新/修正/删除`；
- [x] 我已确认英文原文（如 en_us.json）存在且完整，内容与中文对应；
- [x] 我已确认提交文件的**路径**和**名称**均**正确**（[例子](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md#提交文件路径的例子)）；
  - 如果是 1.12 翻译，应该是：projects/1.12.2/assets/{CurseForge 项目名称}/{ModID}/lang/zh_cn.lang
  - 如果是 1.16 及以上的翻译，应该是：projects/{版本}/assets/{CurseForge 项目名称}/{ModID}/lang/zh_cn.json
- [x] 我已阅读并同意按 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh) 协议发布我的作品；
- [ ] 刷新 PR 的标签/状态，有需要再点击；

祝审核们好运（）

### CODE REVIEW

**Comment by ChuijkYahus** [comment]
/diff
让我看看bot

**Comment by ChuijkYahus** [comment]
<details>
<summary><b>石材</b>（部分名称中本就有砖的视情况而定）</summary>

英文名|中文名
-:|:-
Polished xx|磨制xx
Tiny Brick Bordered xx|微型砖沿xx
Engraved xx|雕纹xx
Centered xx|环纹xx
Unamused xx Carving|不悦徽xx浮雕
Inlaid xx|嵌花xx
Rough xx|粗糙xx
Laid xx Bricks|铺砌xx砖
Pillar xx Bricks|柱式xx砖
Tiny Layered xx Bricks|密层xx砖
Spider xx Carving|蜘蛛徽xx浮雕
Duh xx Carving|惊讶徽xx浮雕（与陶瓦统一）
Brick Bordered xx|砖沿xx
Offset xx Bricks|错排xx砖
xx Bricks|xx砖
xx Mini Tiles|xx小型瓦
Sad xx Carving|悲伤徽xx浮雕
Crying xx Carving|愤怒徽xx浮雕
Cut Blank xx|空白式xx浮雕（与陶瓦统一）
Cut Glad xx|欢愉徽xx浮雕（与陶瓦统一）
Cobbled xx|圆石纹xx
Edged xx Bricks|刻沿xx砖
Creeper xx Carving|苦力怕徽xx浮雕
Eroded xx|风蚀xx
Etched xx Bricks|蚀刻xx砖
Flat xx Tiles|平整xx瓦
Thick Inlaid xx|厚沿嵌花xx
Inscribed xx|纂刻xx
Bordered xx|带框xx
Loded xx|磁石纹xx
Cracked Disordered xx Bricks|裂纹乱砌xx砖
Overlapping xx Tiles|复纹xx
Cracked xx Bricks|裂纹xx砖
Cracked Flat xx Tiles|裂纹平整xx瓦
Prismal xx|海晶纹xx
Blank xx Carving|未刻xx浮雕
Rounded xx Bricks|圆角xx砖
Runic xx Carving|符文xx浮雕
Cut xx Column|切制xx柱石
Sanded xx|沙纹xx
Small xx Bricks|xx小型砖
Smooth xx Column|平滑xx柱石
Smooth Inlaid xx|平滑嵌花xx
Smooth Ringed xx|平滑回纹xx
Smooth Double Inlaid xx|平滑双重嵌花xx
Checkered xx Tiles|棋盘格xx瓦
Spiraled xx|螺旋纹xx
Stacked xx Bricks|叠层xx砖
Angry xx Carving|失望徽xx浮雕（与陶瓦统一）
Tiled xx|方纹xx
Tiled xx Column|方纹xx柱石
Tiled Bordered xx|方纹带框xx
Tiny xx Bricks|xx微型砖
Carved xx|刻制xx
Tiny Layered xx Slabs|细层xx砖
Simple xx Pillar|普通xx柱
Trodden xx|踏裂xx
xx Scales|xx鳞块
Vertical Cut xx|竖纹切制xx
Vertical Disordered xx Bricks|竖纹乱砌xx砖
Weathered xx|风化xx
xx Big Bricks|xx大型砖
Curly xx Pillar|螺卷纹xx柱
Fine xx Pillar|精细xx柱
Simple xx Pillar|普通xx柱
Curvy xx|回索纹xx
Rocky xx|岩状xx
Fancy xx Pillar|精致xx柱
Layered xx Pillar|层纹xx柱

</details>
<details>
<summary><b>陶瓦、黏土</b>（黏土使用“瓦”）</summary>

英文名|中文名
-:|:-
xx Tiles|xx瓦砖
Uneven xx Tiles|错砌xx瓦砖
Uneven xx Bricks|错砌xx砖
Edged xx|刻沿xx
Edged xx Pillar|刻沿xx柱
Fancy xx Pillar|精致xx柱
Chiseled xx|錾制xx
Triangle xx|三角纹xx
Smooth xx|平滑xx
Circular xx|圆形纹xx
Inscribed xx|纂刻xx
Rounded xx Tiles|圆角xx瓦砖
Vertical xx|竖纹xx
Channeled xx|槽纹xx
Embedded xx Tiles|嵌纹xx瓦砖
Fancy xx Tiles|精致xx瓦砖
Fancy Connected xx|精致连缀xx
Diamond xx|菱形纹xx
Plain xx Pillar|素纹xx柱
Centered xx|环纹xx
Chiseled xx|錾制xx
Square xx|方格纹xx
Tiny Bordered xx|小型砖沿xx
Brick Bordered xx|砖沿xx
Medium Bordered xx|中厚沿xx
Squiggle xx|蜿线纹xx
Pebble xx|鹅卵纹xx
Stacked xx|叠层xx
Carved xx|刻制xx
Vertical xx Tiles|竖纹xx瓦砖
Vertical Stacked xx|竖纹叠层xx
Creeper xx|苦力怕徽xx
Thick Square xx|厚方纹xx
Happy xx|欢愉徽xx
Flat xx|空白式xx
Disappointed xx|失望徽xx
Sad xx|悲伤徽xx
Angry xx|愤怒徽xx
Dungeons xx|蜘蛛徽xx（与石材统一）
Surprised xx|惊讶徽xx
Unamused xx|不悦徽xx
Wavy xx|浪纹xx
Inlaid xx|嵌花xx
Decorative xx|装饰性xx
Prismal xx|海晶纹xx
Mini xx Tiles|迷你xx瓦砖
Thick Inlaid xx|厚沿嵌花xx
Checkered xx|棋盘格xx
Flat Bordered xx|细沿xx
Scaled xx|鳞纹xx
Medium xx Bricks|xx中型砖
Vertical xx Bricks|竖纹xx砖
Tiny Vertical xx Bricks|细竖纹xx砖
xx Bricks|xx砖
Cracked xx Bricks|裂纹xx砖
Offset xx Bricks|错排xx砖
Small xx Bricks|xx小型砖
Tiny xx Bricks|xx微型砖
Mixed xx Bricks|混合式xx砖
Thick xx Bricks|厚方xx砖
Inlaid xx Bricks|嵌花xx砖
Bordered xx|带框xx
Mural xx|壁刻xx
Inscription xx|镌刻xx
Big xx Bricks|xx大型砖
Diamond Mural xx|菱形壁刻xx

</details>
<details>
<summary><b>带釉陶瓦</b></summary>

英文名|中文名
-:|:-
Ringed xx|回纹xx
Sphere xx|圆形纹xx
xx Smooth Tiles|xx平滑瓦砖
xx Tiles|xx瓦砖
Decorated xx|饰纹xx
Hexagonical xx|六角纹xx
Minimized xx|迷你纹xx
Paved xx|铺饰xx
Shrunken xx|微缩纹xx
Starry xx|星缀xx
Waved xx|浪纹xx

</details>
<details>
<summary><b>染色玻璃、染色玻璃板</b></summary>

英文名|中文名
-:|:-
Circular xx|圆形纹
Fancy xx|精致xx
Large cc Diamond xx|大菱形纹xx
Ornate xx|精饰xx
Raster xx|栅格xx
cc Diamond xx|菱形纹xx
Square xx|方格xx
Tiled xx|方纹xx
Vertical Striped xx|竖纹xx
Woven xx|编织纹xx

</details>
<details>
<summary><b>混凝土</b></summary>

英文名|中文名
-:|:-
Grid xx|格纹xx
xx Bricks|xx砖
xx Tiles|xx瓦
Stacked xx|叠层xx
Smooth xx|平滑xx
Pegged xx|桩钉xx
xx Panel|xx方板
Creeper xx|苦力怕徽xx
Warning xx|警告xx
Hazard xx|警示xx
Rusted xx|锈板xx
Reinforced xx|强化xx
Bolted xx|螺栓xx
Railed xx|箍装xx
Grill xx|铁网xx
Wired xx|装线xx
Stop xx|禁止符xx
Mosaic xx|马赛克xx
Striped xx|斜纹xx
xx Pillar|xx柱
Framed xx Panel|带框xx方板

</details>
<details>
<summary><b>羊毛、地毯</b></summary>

英文名|中文名
-:|:-
Bark xx|层织xx
Blocky xx|砖织xx
Bundled xx|扎织xx
Cornered xx|回织xx
Crafted xx|框织xx
Crosshaired xx|十字织xx
Cubed xx|正方织xx
Harsh Quilted xx|密织xx
Knitted xx|针织xx
Patched xx|缝织xx
Rectangular xx|长方织xx
Rippled xx|浪纹xx
Scaled xx|鳞织xx
Soft xx|柔顺xx
Stringed xx|穿线xx
Tiled xx|方格xx
Tripped xx|绳织xx
Wickered xx|柳条织xx
Windmill xx|风车织xx
Woved xx|编织xx

</details>
<details>
<summary><b>木板</b></summary>

英文名|中文名
-:|:-
Stacked xx|叠层xx
Hewn xx|劈纹xx
Vertically Railed xx|竖式包栏纹xx
Framed xx|框纹xx
Detailed xx|精刻xx
Bricky xx|砖纹xx
Cornered xx|包角纹xx
Wickered xx|柳编纹xx
Shifted xx|错编纹xx
Thin xx|细纹xx
Enclosed xx|包纹xx
Boxed xx|箱纹xx
Crated xx|板条箱纹xx
Polished xx|磨制xx
Vertical xx|竖纹xx
Smooth xx|平滑xx
Diagonal xx|斜纹xx
Basket Woven xx|篮编纹xx
cc Shavings|cc薄木板
Railed xx|包栏纹xx
Mosaic xx|马赛克xx
cc Panel|cc方木板
Brick Bond xx|砖式xx
Cross Laced xx|交编纹xx
Crossed xx|交织纹xx
Diamond xx|菱形纹xx
Double Herringbone xx|双重箭尾纹xx
Fine xx|精纹xx
Fine Vertical xx|竖精纹xx
Herringbone xx|箭尾纹xx
Laced xx|穿编纹xx
Nailed xx|饰钉xx
Natural xx|自然纹xx
Pegged xx|桩钉xx
Slanted xx|斜式xx
Tiled xx|方纹xx
Versailles xx|凡尔赛纹xx
Whirlwind xx|风车纹xx
cc Framed Board|cc带框木板
Zigzag xx|之字纹xx
Lattice xx|回编纹xx

</details>
<details>
<summary><b>树叶</b></summary>

英文名|中文名
-:|:-
Apple xx|苹果xx
Cherry xx|樱桃xx
Golden Cherry xx|金樱桃xx
Golden Apple xx|金苹果xx
Flowering xx|盛开的xx
Azalea xx|杜鹃花xx
Yellow xx|黄色xx
Orange xx|橙色xx
Red xx|红色xx
Brown xx|棕色xx
White xx|白色xx
Snowy xx|覆雪xx

</details>
<details>
<summary><b>原木</b></summary>

英文名|中文名
-:|:-
Chipped xx|凿制xx
Firewood xx|xx木柴捆
Center Cut xx|中央去皮xx
Damaged xx|破损的xx
Edge Cut xx|边缘去皮xx
Flowering xx|开花的xx
Mixed xx|混合式xx
Nailed xx|饰钉xx
Overgrown xx|缠蔓xx
Planked xx|饰板xx
Reinforced xx|强化xx

</details>
<details>
<summary><b>去皮原木</b>（目前仅诡异）</summary>

英文名|中文名
-:|:-
Carved xx|刻制xx
Chipped xx|凿制xx
D Sign xx|D形刻记xx
Edged xx|刻沿xx
F Sign xx|F形刻记xx
I Sign xx|I形刻记xx
K Sign xx|K形刻记xx
Knotted xx|生节xx
L Sign xx|L形刻记xx
Lumpy xx|生瘤xx
Layered xx|层纹xx
M Sign xx|M形刻记xx
Patient xx|耐心徽xx
Reinforced xx|强化xx
xx Corner|xx转角
Sign xx|刻记xx
Wise xx|明智徽xx
Stern xx|苛刻徽xx
Stacked xx|叠层xx

</details>
<details>
<summary><b>木门</b></summary>

英文名|中文名
-:|:-
cc Slide Door|cc推拉门
Beach xx|海滨式xx
Boarded xx|格板式xx
Dual Paneled xx|双格式xx
Fortified xx|加固xx
Gated xx|四格式xx
Glass xx|玻璃xx
Heavy xx|厚格式xx
Modern xx|现代式xx
Overgrown xx|缠蔓xx
Paneled xx|格板式xx
Paper xx|纸障式xx
Pressed xx|压板式xx
Screen xx|竖屏式xx
Secret xx|隐秘式xx
Shack xx|陋屋式xx
Supported xx|横板式xx
Tile Windowed xx|带窗格xx
Tiled xx|方格式xx
Windowed xx|带窗xx

</details>
<details>
<summary><b>木桶</b></summary>

英文名|中文名
-:|:-
Acacia Barrel|金合欢木桶
Acacia Crate|金合欢木板条箱
Bamboo Barrel|竹桶
Birch Barrel|白桦木桶
Birch Crate|白桦木板条箱
Coal Barrel|装满煤炭的木桶
Copper Barrel|装满铜的木桶
Crimson Barrel|绯红木桶
Crimson Crate|绯红木板条箱
Fish Barrel|装满鱼的木桶
Fruit Barrel|装满水果的木桶
Gilded Barrel|镶金木桶
Dark Oak Crate|深色橡木板条箱
Gold Barrel|装满金的木桶
Iron Barrel|装满铁的木桶
Jungle Crate|丛林木板条箱
Metal Barrel|金属桶
Mushroom Barrel|蘑菇桶
Oak Barrel|橡木桶
Oak Crate|橡木板条箱
Reinforced Acacia Crate|金合欢木加固板条箱
Netherite Safe|下界合金保险箱
Reinforced Birch Crate|白桦木加固板条箱
Reinforced Crimson Crate|绯红木加固板条箱
Reinforced Dark Oak Crate|深色橡木加固板条箱
Reinforced Jungle Crate|丛林木加固板条箱
Reinforced Oak Crate|橡木加固板条箱
Reinforced Spruce Crate|云杉木加固板条箱
Reinforced Warped Crate|诡异木加固板条箱
Spruce Barrel|云杉木桶
Spruce Crate|云杉木板条箱
Varnished Barrel|涂漆木桶
Warped Crate|诡异木板条箱

</details>
<details>
<summary><b>矿物块</b></summary>

英文名|中文名
-:|:-
Sheet xx|叠板xx
Panel xx|方格xx
Embossed xx|压纹xx
Pressed xx|压制xx
Reinforced xx|强化xx
Stacked xx|叠层xx
Layered xx|层纹xx
Shuttered xx|窗格xx
Chipped xx|凿制xx
Engraved xx|雕纹xx
Plated xx|宽板xx
Ancient xx|古式xx

</details>
<details>
<summary><b>铁栏杆</b></summary>

英文名|中文名
-:|:-
Zippered Iron Bars|拉链式铁栏杆
Barbed Iron Bars|倒钩式铁栏杆
Bolted Iron Bars|栓接式铁栏杆
Chained Iron Bars|链连式铁栏杆
Crossbolted Iron Bars|交叉栓接式铁栏杆
Crossed Iron Bars|交叉式铁栏杆
Floral Iron Bars|纹花式铁栏杆
Harp Iron Bars|竖琴式铁栏杆
Single Iron Bar|单柱式铁栏杆
Linked Iron Bars|连列式铁栏杆
Loose Iron Bars|散列式铁栏杆
Piked Iron Bars|矛尖式铁栏杆
Railed Iron Bars|横栏式铁栏杆
Sighthole Iron Bars|圆孔式铁栏杆
Stepped Iron Bars|梯式铁栏杆
Sturdy Iron Bars|坚固铁栏杆
Supported Iron Bars|横杆式铁栏杆
Thorned Iron Bars|带刺式铁栏杆
Victorian Iron Bars|维多利亚式铁栏杆
Vined Iron Bars|纹藤式铁栏杆
Woven Iron Bars|编织式铁栏杆

</details>
<details>
<summary><b>蘑菇、菌类</b></summary>

英文名|中文名
-:|:-
Triple xx|三重xx
Triple Wizard xx|三重尖顶xx
Duo xx|双重xx
Curved xx|曲盖xx
Rounded xx|圆盖xx
Mismatched xx|错生xx
Duo Wizard xx|双重尖顶xx
Button xx|圆顶xx
Wizard xx|尖顶xx
Duo Button xx|双重圆顶xx
Teeny xx|小型xx
Tiny xx|迷你xx
Scattered xx|散生xx
Mini Scattered xx|散生迷你xx
Single xx|单生xx

</details>
<details>
<summary><b>蜘蛛网</b></summary>

英文名|中文名
-:|:-
Floor Cobweb|覆地蜘蛛网
Stretched Cobweb|拉伸蜘蛛网
Dense Cobweb|细密蜘蛛网
Broken Hanging Cobweb|破裂的垂挂蜘蛛网
Scattered Cobweb|散生蜘蛛网
Thick Floor Cobweb|覆地厚蜘蛛网
Hanging Cobweb|垂挂蜘蛛网
Arch Cobweb|拱形蜘蛛网
Stringy Cobweb|络织蜘蛛网
Broken Cobweb|破裂的蜘蛛网

</details>
<details>
<summary><b>骨块</b></summary>

英文名|中文名
-:|:-
Chiseled Bone Block|錾制骨块
Cracked Bone Block|裂纹骨块
Etched Bone Block|蚀刻骨块
Aged Bone Block|古式骨块
Wrapped Bone Block|缠绳骨块
Inscribed Bone Block|纂刻骨块
Tiled Bone Block|方纹骨块
Reinforced Bone Block|强化骨块
Bone Pillar|骨柱
Windmill Bone Block|风车纹骨块
Bound Bone Block|绑带骨块

</details>
<details>
<summary><b>骨块</b></summary>

英文名|中文名
-:|:-
Messy Bookshelf|凌乱书架
Scholar's Bookshelf|学者书架
Mage's Bookshelf|法师书架
Dusty Bookshelf|积尘书架
Professor's Bookshelf|教授书架
Creepy Bookshelf|怪异书架
Explorer's Bookshelf|探索者书架
Organized Bookshelf|整洁书架
Botanist's Bookshelf|植物学家书架
Florist's Bookshelf|花匠书架
Nether Bookshelf|下界书架
Empty Bookshelf|空书架
Vertical Bookshelf|竖式书架
Biologist's Bookshelf|生物学家书架
Nature Bookshelf|自然式书架
Soul Bookshelf|灵魂书架

</details>
<details>
<summary><b>蘑菇柄</b></summary>

英文名|中文名
-:|:-
Broken Mushroom Stem|破裂的蘑菇柄
Dark Mushroom Stem|暗色蘑菇柄
Groveled Mushroom Stem|倒伏蘑菇柄
Scaled Mushroom Stem|鳞纹蘑菇柄
Smooth Mushroom Stem|平滑蘑菇柄
Spreckled Mushroom Stem|斑点蘑菇柄
Stripped Mushroom Stem|去皮蘑菇柄

</details>

今天先到这，明天继续

**Comment by IdealNightOcean** [comment]
我进游戏看了一下。
`Center Cut xx`是中间有树皮，两侧没有，而`中央去皮xx`按一般理解是“中央的皮被去掉的xx”；
`Edge Cut xx`是两侧有树皮，中间没有，问题也一样；
我觉得这里中文应该和英文相反翻译，将`Center Cut xx`翻译为“边缘去皮xx”，将`Edge Cut xx`翻译为“边缘去皮xx”。

File: projects/1.18/assets/chipped/chipped/lang/zh_cn.json [hunk 1]
```diff
@@ -1,2510 +1,7786 @@
...
+  	"block.chipped.granite_48": "叠层花岗岩砖",
+  	"block.chipped.granite_49": "失望徽花岗岩浮雕",
+  	"block.chipped.granite_50": "方纹花岗岩",
+  	"block.chipped.granite_51": "方纹花岗岩柱石",
+  	"block.chipped.granite_52": "方纹带框花岗岩",
+  	"block.chipped.granite_53": "花岗岩微型砖",
+  	"block.chipped.granite_54": "刻制花岗岩",
+  	"block.chipped.granite_55": "细层花岗岩砖",
+  	"block.chipped.granite_56": "普通花岗岩柱",
+  	"block.chipped.gra
```
Comment (IdealNightOcean) [inline projects/1.18/assets/chipped/chipped/lang/zh_cn.json:3658]:
```suggestion
    "block.chipped.ladder_10": "坚韧梯子",
```

Comment (IdealNightOcean) [reply projects/1.18/assets/chipped/chipped/lang/zh_cn.json:3658]: 这预览不知道跑哪去了，suggestion在L3658

Comment (IdealNightOcean) [inline projects/1.18/assets/chipped/chipped/lang/zh_cn.json:6547]:
```suggestion
	"block.chipped.sand_2": "粗沙",
```

Comment (IdealNightOcean) [inline projects/1.18/assets/chipped/chipped/lang/zh_cn.json:6554]:
```suggestion
	"block.chipped.sand_9": "细沙",
```

**Comment by IdealNightOcean** [comment]
因为‘xx Tiles’翻译为“xx瓦砖”，`xx Bricks`翻译成“xx砖”，所以出现了大量的“xx陶瓦瓦砖”和“xx陶瓦砖”，看起来有点怪。
可以考虑把‘xx Tiles’翻译为“xx瓷砖”，这样就能有效区分“xx陶瓦瓷砖”和“xx陶瓦砖”了。
只是建议~~~

**Review by IdealNightOcean — CHANGES_REQUESTED** [review]

File: projects/1.18/assets/chipped/chipped/lang/zh_cn.json (hunk 1 above)
Comment (IdealNightOcean) [reply projects/1.18/assets/chipped/chipped/lang/zh_cn.json:3658]: 预览有点问题，担心的话就手动改吧

**Comment by IdealNightOcean** [comment]
文件结尾的几十行空行删了吧，反正留着没用

**Comment by ChuijkYahus** [comment]
> 我进游戏看了一下。 `Center Cut xx`是中间有树皮，两侧没有，而`中央去皮xx`按一般理解是“中央的皮被去掉的xx”； `Edge Cut xx`是两侧有树皮，中间没有，问题也一样； 我觉得这里中文应该和英文相反翻译，将`Center Cut xx`翻译为“边缘去皮xx”，将`Edge Cut xx`翻译为“中央去皮xx”。

这里要不整成`中央切制`和`边缘切制`

---

> 因为`xx Tiles`翻译为“xx瓦砖”，`xx Bricks`翻译成“xx砖”，所以出现了大量的“xx陶瓦瓦砖”和“xx陶瓦砖”，看起来有点怪。 可以考虑把`xx Tiles`翻译为“xx瓷砖”，这样就能有效区分“xx陶瓦瓷砖”和“xx陶瓦砖”了。 只是建议~~~

之前其实一直想要的是`陶瓦瓦`，主要是`Tiles`在原版的翻译为`瓦`
如果要改不知道`方砖`可不可以？这里的实际材质是田字格，`Tiled`在这模组里也一般叫`方纹`

**Comment by ChuijkYahus** [comment]
> 文件结尾的几十行空行删了吧，反正留着没用

这个可以不用管的，formatter会帮忙删掉

**Comment by IdealNightOcean** [comment]
大问题，门的翻译是乱掉的，名字和模型对不上。

**Comment by ChuijkYahus** [comment]
> 大问题，门的翻译是乱掉的，名字和模型对不上。可能是mod版本问题？

模组本身存在材质类型和名称不对应的问题，官库那边也有issue
挺头疼的

**Comment by IdealNightOcean** [comment]
> 模组本身存在材质类型和名称不对应的问题，官库那边也有issue
> 挺头疼的

挺好，那就是他们的问题了

**Comment by IdealNightOcean** [comment]
> > 我进游戏看了一下。 `Center Cut xx`是中间有树皮，两侧没有，而`中央去皮xx`按一般理解是“中央的皮被去掉的xx”； `Edge Cut xx`是两侧有树皮，中间没有，问题也一样； 我觉得这里中文应该和英文相反翻译，将`Center Cut xx`翻译为“边缘去皮xx”，将`Edge Cut xx`翻译为“中央去皮xx”。
> 
> 这里要不整成`中央切制`和`边缘切制`
> 
> > 因为`xx Tiles`翻译为“xx瓦砖”，`xx Bricks`翻译成“xx砖”，所以出现了大量的“xx陶瓦瓦砖”和“xx陶瓦砖”，看起来有点怪。 可以考虑把`xx Tiles`翻译为“xx瓷砖”，这样就能有效区分“xx陶瓦瓷砖”和“xx陶瓦砖”了。 只是建议~~~
> 
> 之前其实一直想要的是`陶瓦瓦`，主要是`Tiles`在原版的翻译为`瓦` 如果要改不知道`方砖`可不可以？这里的实际材质是田字格，`Tiled`在这模组里也一般叫`方纹`

还是叫`去皮`吧，毕竟材质就是去了一半皮的原木。
`Tiles`不好办，其他的`方纹xx`没用连接纹理问题，但是`Tiles`是有连接纹理的，连着铺几块就不是方纹了。

**Comment by ChuijkYahus** [comment]
> > > 我进游戏看了一下。 `Center Cut xx`是中间有树皮，两侧没有，而`中央去皮xx`按一般理解是“中央的皮被去掉的xx”； `Edge Cut xx`是两侧有树皮，中间没有，问题也一样； 我觉得这里中文应该和英文相反翻译，将`Center Cut xx`翻译为“边缘去皮xx”，将`Edge Cut xx`翻译为“中央去皮xx”。
> > 
> > 
> > 这里要不整成`中央切制`和`边缘切制`
> > > 因为`xx Tiles`翻译为“xx瓦砖”，`xx Bricks`翻译成“xx砖”，所以出现了大量的“xx陶瓦瓦砖”和“xx陶瓦砖”，看起来有点怪。 可以考虑把`xx Tiles`翻译为“xx瓷砖”，这样就能有效区分“xx陶瓦瓷砖”和“xx陶瓦砖”了。 只是建议~~~
> > 
> > 
> > 之前其实一直想要的是`陶瓦瓦`，主要是`Tiles`在原版的翻译为`瓦` 如果要改不知道`方砖`可不可以？这里的实际材质是田字格，`Tiled`在这模组里也一般叫`方纹`
> 
> 还是叫`去皮`吧，毕竟材质就是去了一半皮的原木。 `Tiles`不好办，其他的`方纹xx`没有连接纹理问题，但是`Tiles`是有连接纹理的，连着铺几块就不是方纹了。


这个应该没有连接纹理吧……

**Comment by IdealNightOcean** [comment]
等一下，我发现你“去皮XX”翻译的没问题，问题本质是mod名称对应混乱，它的`Center Cut xx`对应了几种不同的材质。~~怪不得~~
`Tiles`
这mod语言文件问题挺让人头大，已经尾大不掉了，不知道制作组会怎么处理

**Comment by IdealNightOcean** [comment]
`Tiles`是我搞错了，只有几种有连接纹理

**Comment by ChuijkYahus** [comment]
> 等一下，我发现你“去皮XX”翻译的没问题，问题本质是mod名称对应混乱，它的`Center Cut xx`对应了几种不同的材质。~怪不得~ 这mod语言文件问题挺让人头大，已经尾大不掉了，不知道制作组会怎么处理

还好手慢
先放着吧，等制作组出下一个版本

**Comment by IdealNightOcean** [comment]
OK，lang本身问题确实大，原木也有名称错位的。只能等着了(；′⌒`)

**Comment by ChuijkYahus** [comment]
> OK，lang本身问题确实大，原木也有名称错位的。只能等着了(；′⌒`)

不过那张表里比较独特的方块（干草块、西瓜这类）基本上都不是错位的
~~因为错位了也看不出来~~

File: projects/1.18/assets/chipped/chipped/lang/zh_cn.json (hunk 1 above)
Comment (Rinko1231) [inline projects/1.18/assets/chipped/chipped/lang/zh_cn.json:4279]:
```suggestion
  	"block.chipped.lodestone_28": "篆刻磁石",
```
`纂`是`编`的意思
看原文应该是篆刻的`篆`(zhuan)。

**Review by Rinko1231 — CHANGES_REQUESTED** [review]
纂和篆，不是同一个字啊（

**Comment by ChuijkYahus** [comment]
> 纂和篆，不是同一个字啊（ `纂`是`编`的意思 看原文inscribed应该是篆刻的`篆`(zhuan)。

替换错了，已改

**Comment by ChuijkYahus** [comment]
作者说1.18不改了（）

**Review by SlimeSB — APPROVED** [review]

**Comment by SlimeSB** [comment]
二次校对后24h 或 无条件72h后可合并

**Pull request merged**